infrastructure/lib/load-schema.ts

```diff
@@ -2,17 +2,25 @@ import * as fs from "fs";
 import * as yaml from "js-yaml";
 import toJsonSchema from "@openapi-contrib/openapi-schema-to-json-schema";
 import * as utils from "./util";
-export function convertSchema(): void {
+import $RefParser from "@apidevtools/json-schema-ref-parser";
+
+export async function convertSchema(): Promise<void> {
   try {
+    const parser = new $RefParser();
```

Comment:
Maybe I am missing something, but I thought we already made the changes for the `$RefParser` in [PR #372](https://github.com/dod-ccpo/atat-web-api/pull/372/files)?